src/lib/logger.js

```
@@ -0,0 +1,161 @@
+const LOG_LEVEL_NAMES = {
+  0: 'ERROR',
+  1: 'WARN',
+  2: 'INFO',
+  3: 'DEBUG',
+  4: 'TRACE'
+};
+
+class Logger {
+  constructor() {
+    this.level = this.getLogLevel();
+    this.isProduction = process.env.NODE_ENV === 'production';
```

ESLint

[error] 'process' is not defined.

(no-undef)